src/registrar/templates/includes/footer.html

```diff
@@ -0,0 +1,123 @@
+{% load static %}
+
+<footer class="usa-footer">
+    <div class="usa-footer__secondary-section">
+      <div class="grid-container">
+        <div class="grid-row grid-gap">
+          <div
+            class="
+              usa-footer__logo
+              grid-row
+              mobile-lg:grid-col-6 mobile-lg:grid-gap-2
+            "
+          >
+            <div class="mobile-lg:grid-col-auto">
+              <img
+                class="usa-footer__logo-img"
+                src="{% static 'img/dottedgov-round.svg' %}"
+                alt="dot gov registrar"
+								width="100px"
+              />
+            </div>
+          </div>
+          <div class="usa-footer__contact-links mobile-lg:grid-col-6">
+            <p class="usa-footer__contact-heading">
+              Contact us
+            </p>
+            <address class="usa-footer__address">
+              <div class="usa-footer__contact-info grid-row grid-gap">
+                <div class="grid-col-auto">
+                  <a href="mailto:registrar@dotgov.gov">registrar@dotgov.gov</a>
+                </div>
+              </div>
+            </address>
+          </div>
+        </div>
+      </div>
+    </div>
+  </footer>
+  <div class="usa-identifier">
+    <section
+      class="usa-identifier__section usa-identifier__section--masthead"
+      aria-label="Agency identifier"
+    >
+      <div class="usa-identifier__container">
+        <div class="usa-identifier__logos">
+          <a href="https://www.cisa.gov" class="usa-identifier__logo"
+            ><img
+              class="usa-identifier__logo-img"
+              src="{% static 'img/CISA_logo.png' %}"
+              alt="CISA logo"
+              role="img"
```

Comment:
```suggestion
              role="img"
              width="200px"
```

**Suggestion (non-blocking)**: without CSS, the logo takes up the majority of the page; setting explicit width will fix that.